compiler/rustc_feature/src/removed.rs

```diff
@@ -271,6 +271,9 @@ declare_features! (
     /// Allows `#[link(kind = "static-nobundle", ...)]`.
     (removed, static_nobundle, "1.63.0", Some(37403),
      Some(r#"subsumed by `#[link(kind = "static", modifiers = "-bundle", ...)]`"#), 95818),
+    /// Allows string patterns to dereference values to match them.
+    (removed, string_deref_patterns, "CURRENT_RUSTC_VERSION", Some(87121),
+        Some("superseded by `deref_patterns`"), 150530),
```

Comment:
Other multi-line entries in this list use an inconsistent mix of 1-space or 4-space indents for their continuation lines.

I chose to use 4 spaces here because it seems easier to read.